docs/admin/node-conformance.md

```diff
@@ -9,54 +9,52 @@ assignees:
 
 ## Node Conformance Test
 
-*Node conformance test* is a test framework validating whether a node meets the
-minimum requirement of Kubernetes with a set of system verification and
-functionality test. A node which passes the tests is qualified to join a
-Kubernetes cluster.
+*Node conformance test* is a containerized test framework validating whether a
+node meets the minimum requirement of Kubernetes with a set of system
+verification and functionality test. A node which passes the test is qualified
+to join a Kubernetes cluster.
 
 ## Limitations
 
 There are following limitations in the current implementation of node
-conformance test. They'll be improved in future version.
+conformance test.
 
 * Node conformance test only supports Docker as the container runtime.
-* Node conformance test doesn't validate network related system configurations
-  and functionalities.
 
 ## Prerequisite
 
 Node conformance test is used to test whether a node is ready to join a
 Kubernetes cluster, so the prerequisite is the same with a standard Kubernetes
-node. At least, the node should have properly installed:
+node. At least, the node should have installed:
 
 * Container Runtime (Docker)
 * Kubelet
 
-Node conformance test validates kernel configurations. If the kenrel module
-`configs` is built as module in your environment, it must be loaded before the
-test. (See [Caveats #3](#caveats) for more information)
-
 ## Usage
 
 ### Run Node Conformance Test
 
 * **Step 1:** Point your Kubelet to localhost `--api-servers="http://localhost:8080"`,
```

Comment:
You can remove "**Step 1:**" and the bullet list, and use a markdown list instead. You can still nest bullets inside of it for the Kubelet flags.

1. Point your Kubelet to localhost, etc.
  * `--pod-cidr`: If you are using `kubenet`...
2. Run the node conformance test with the following command: